tests/foreman/ui/test_organization.py

```diff
@@ -289,3 +324,46 @@ def test_positive_download_debug_cert_after_refresh(session):
                 session.subscription.refresh_manifest()
     finally:
         entities.Subscription(organization=org).delete_manifest(data={'organization_id': org.id})
+
+
+@pytest.mark.tier2
+def test_positive_errata_view_organization_switch(
+    session, module_org, module_lce, module_repos_col, default_sat
+):
+    """Verify no errata list visible on Organization switch
+    :id: a5317b2d-2593-4bee-9b20-78066cf2964c
```

Comment:
Have a new line for each of the docstring so it can be consistent with the entire file.

Reply:
Updated.